spec/robots/dor_repo/ocr/start_ocr_spec.rb

```diff
@@ -36,41 +36,6 @@
     end
   end
 
-  context 'when the object is not opened and it is possible to OCR, but open version fails' do
-    let(:version_open) { false }
-    let(:possible) { true }
-
-    before { allow(Honeybadger).to receive(:notify) }
-
-    context 'when open version fails twice and then succeeds' do
-      before do
-        count = 0
-        allow(version_client).to receive(:open) do |*_args|
-          count += 1
-          raise Dor::Services::Client::UnexpectedResponse.new(response: 'nope') unless count > 2
-
-          true
-        end
-      end
-
-      it 'retries the first two errors, then calls open again and logs to honeybadger' do
-        expect { perform }.not_to raise_error
-        expect(version_client).to have_received(:open).thrice # shakespeare coding (first two times fail, third time succeeds)
-        expect(Honeybadger).to have_received(:notify).twice # two calls to HB for the first two failures
-      end
-    end
-
-    context 'when open version fails and exceeds maximum tries' do
-      before { allow(version_client).to receive(:open).and_raise(Dor::Services::Client::UnexpectedResponse.new(response: 'nope')) }
-
-      it 'logs to honeybadger and then raises the error' do
-        expect { perform }.to raise_error(Dor::Services::Client::UnexpectedResponse)
-        expect(version_client).to have_received(:open).thrice # shakespeare coding (all three times fail)
-        expect(Honeybadger).to have_received(:notify).twice # two calls to HB for the first two failures
-      end
-    end
-  end
-
```

Comment:
these tests are no longer needed here and are instead performed in the new spec for the new `VersionUpdated` class